### ISSUE TITLE
Add usecols option

### ISSUE THREAD

**Issue by pdbaines** [wiseio/paratext#25]
With same functionality as `pandas.read_csv` (see docs [here](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html)).

**Comment by andytwigg** [comment]
Agree, this would be useful. In general, supporting the same args as read_csv would be good, including `sampling` and `nrows`.
